stratum/docs/setup_guide_barefoot_tofino_onos.md

```diff
@@ -0,0 +1,198 @@
+<!--
+Copyright 2020-present Open Networking Foundation
+
+SPDX-License-Identifier: Apache-2.0
+-->
+# Installation guide:<br/>ONOS + Stratum + Barefoot Tofino switch
+
+These instructions have been tested on an [Accton Wedge 100BF-32X](https://www.edge-core.com/productsInfo.php?cls=1&cls2=180&cls3=181&id=335), yet should work on all Tofino-based switches.
+
+The coarse steps are as follows:
+- On host: build ONL
+- On switch: install built ONL
+- On switch: start Stratum
+- On host: start ONOS
+- On host: build `fabric-tofino` ONOS app
+- On host: install and activate `fabric-tofino` app in ONOS
+- On host: register switch in ONOS using `netcfg`
+
+
+## Build ONL
+
+:warning: You have to build ONL - if you use a pre-built installer, you will run into trouble with Stratum.
+
+ONL can be built wherever you like.
+
+```
+git clone --depth 1  --single-branch --branch ONLPv2 git@github.com:opencomputeproject/OpenNetworkLinux.git
```

Comment:
In the meeting, a separate fork of ONL was mentioned which should be used instead of upstream?
Can someone provide the link, please?

Reply:
Here is the link: https://github.com/opennetworkinglab/OpenNetworkLinux
Note that you need to switch to `onf-ONLPv2` branch

We also provide pre-built images here:
https://github.com/opennetworkinglab/OpenNetworkLinux/releases